website/docs/basics/blocks-intro.md

```diff
@@ -21,6 +21,8 @@ For more info about blocks, you can read the [block structure](block-structure)
 
 ### Do I need to have storybook stories in my block?
 
+No you don't. But we provided you an ability to use storybook for all of your blocks and components. Why not use it.
```

Comment:
```suggestion
No, you don't. But we provided you with the ability to use the storybook for all of your blocks and components. Why not use it? It will speed up your development time. Trust us 🙂.
```